common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -276,7 +276,7 @@ def transcript(self, request, dispatch):
 
         elif dispatch.startswith('available_translations'):
 
-            available_translations = self.available_translations(transcripts)
+            available_translations = self.available_translations(transcripts, verify_assets=True)
```

Comment:
Why aren't we falling back to English transcripts here?

Reply:
I don't see any good reason for not having consistency among the usages of `verify_assets`.

_Lets assume_ `FALLBACK_TO_ENGLISH_TRANSCRIPTS=True`
On a fresh load, we won't verify assets and language "English" shows up in lang menu
And, selecting "English" from the lang menu hides it away from that menu (i.e. due to an ajax call to this more strict dispatch)

@OmarIthawi, do you see any issues if we totally depend on feature `FALLBACK_TO_ENGLISH_TRANSCRIPTS`? 
I don't think, there is any point of having a language in the menu whose content is not in place (i.e. not verified)

@muzaffaryousaf I'd like to hear your thoughts on this too.

Reply:
@Qubad786, sorry didn't see your comment earlier. `FALLBACK_TO_ENGLISH_TRANSCRIPTS` should incur a small performance penalty. Which is the point of returning `en` always. Not sure what is the real cost of enabling it at the scale of edX.

But at Edraak we're happy with this verification and didn't notice any penalty.

After all, it's just a feature, so the DevOps team can toggle and monitor it.